test/core/end2end/end2end_tests.h

```diff
@@ -741,6 +747,33 @@ core_end2end_test_fuzzer::Msg ParseTestProto(std::string text);
   CORE_END2END_FUZZER(suite, name)                                             \
   void CoreEnd2endTest_##suite##_##name::RunTest()
 
+#define CORE_END2END_TEST_INCOMPATIBLE_WITH_FUZZING(suite, name)               \
+  class CoreEnd2endTest_##suite##_##name final                                 \
+      : public grpc_core::CoreEnd2endTest {                                    \
+   public:                                                                     \
+    using grpc_core::CoreEnd2endTest::CoreEnd2endTest;                         \
+    void RunTest();                                                            \
+  };                                                                           \
+  void suite##_##name(const grpc_core::CoreTestConfiguration* config,          \
+                      core_end2end_test_fuzzer::Msg msg) {                     \
+    if (config == nullptr) {                                                   \
+      GTEST_SKIP() << "config not available on this platform";                 \
+    }                                                                          \
+    if (absl::StartsWith(#name, "DISABLED_")) GTEST_SKIP() << "disabled test"; \
+    if (!IsEventEngineListenerEnabled() || !IsEventEngineClientEnabled() ||    \
```

Comment:
Presumably, we don't need the checks on line 763 or 767, since we're not fuzzing here?

Reply:
we can remove the entire enclosing function even!